cosi/cosi.go

```diff
@@ -221,80 +227,97 @@ func (c *CoSi) VerifyResponses(aggregatedPublic abstract.Point) error {
 	left := c.suite.Point().Add(kA, sB)
 
 	if !left.Equal(c.aggregateCommitment) {
-		return errors.New("recreated commitment is not equal to one given")
+		return errors.New("recreated commitment is not equal to the given one")
 	}
 
 	return nil
 }
 
-// VerifySignature is the method to call to verify a signature issued by a Cosi
-// struct. Publics is the WHOLE list of publics keys, the mask at the end of the
-// signature will take care of removing the indivual public keys that did not
-// participate
+// Verify checks the given cosignature using the provided signing policy.
+func Verify(suite abstract.Suite, publics []abstract.Point, policy Policy, message, sig []byte) error {
```

Comment:
I would prefer to give the policy as the last argument. The reason for that is to be able to quickly and easily verify any signature without necessarily holding a `Policy` object. Having the "nil" argument in the middle is cumbersome because you need to remember exactly at which position is the policy one.
Pushing the idea further, if no policy is given, maybe use a default policy requiring at least one or two signers.

In short, movin from 
```go
// definition
func Verify(suite abstract.Suite, publics []abstract.Point, policy Policy, message, sig []byte) error 
// calling
cosi.Verify(suite,publics,nil,msg,sig)
```
to
```go
// definition
func Verify(suite abstract.Suite, publics []abstract.Point, message, sig []byte, policy Policy) error 
// calling
cosi.Verify(suite,publics,msg,sig,nil)
```
 What do you think about it ?